packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -591,6 +591,11 @@ class WebSettings extends JavaObject {
     return api.setAllowFileAccessFromInstance(this, enabled);
   }
 
+  /// Sets the text zoom of the page in percent. The default is 100.
```

Comment:
There should be an empty space after the first sentence: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
fixed with: 14223ee4ba9c3ba02046d6efd78be1a05868ec0f

Reply:
The second sentence is still included in the first line. `The default is 100.` needs to move down two lines.

Reply:
Oops! Updated with 763bd28237e4d2e19ed2ca7ec8fdf4671d88223b